lib/functions/losses.ts

```diff
@@ -16,7 +16,7 @@ function meanSquaredError(prediction: number[], target: number[]): number {
         value += (prediction[i] - target[i]) ** 2
     }
 
-    return value / target.length
+    return value / (2 * target.length)
```

Comment:
why 2*?

Reply:
Dividing the expression by 2 will make the derivative calculations simpler. 
From `-2 * input * (target - prediction)` to `-1 * input * (target - prediction)`